templates/index.html

```diff
@@ -1,63 +1,14 @@
-      </div>
-
-      {% block content %}
-      <p class="content__title__text">
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate in
-        ad earum iusto perferendis necessitatibus, harum voluptatibus incidunt
-        consectetur qui maxime eligendi tenetur odit tempora rerum et quo
-        itaque. Iusto enim blanditiis consectetur nisi maxime in, quas fugit
-        dolores similique dolorem cupiditate, officia obcaecati molestiae totam.
-        Delectus, repellat ducimus consectetur ullam corporis nisi, ratione cum
-        in quis, eum deleniti. Architecto adipisci, sit illum nostrum quas
-        perferendis facere recusandae itaque est obcaecati fugit provident
-        aliquid, ab suscipit sapiente odio doloremque unde natus! Nobis
-        cupiditate omnis ex perspiciatis explicabo natus nostrum modi,
-        exercitationem unde, perferendis, vero qui commodi accusantium
-        doloremque. Eos illum rerum ea libero tempora. In facilis veniam
-        obcaecati quisquam sed labore impedit. Consectetur, voluptatibus error
-        a, rem debitis recusandae minus culpa suscipit officia illum ab itaque!
-        Cumque minima aspernatur debitis perferendis mollitia voluptas rem
-        fugiat quasi dicta ipsam, inventore vel fuga atque quae voluptatibus
-        perspiciatis, incidunt quaerat ducimus soluta eaque praesentium adipisci
-        reiciendis. Ipsum quo voluptatibus maxime, temporibus nemo numquam hic
-        dolores molestiae qui, sapiente quos aperiam exercitationem quod
-        consectetur consequatur dolorum. Natus deserunt molestiae, magni
-        expedita a, quo saepe facilis fugit pariatur ea est sit? Ea, eum
-        perspiciatis ratione tempore nostrum explicabo mollitia ipsa delectus
-        architecto dolore. Id dignissimos eum, dolorem optio ut magni pariatur
-        minus eligendi inventore in quidem repudiandae facilis odio tenetur aut
-        earum velit ullam enim dolor beatae laboriosam, amet eos ea? Blanditiis
-        harum voluptas, aliquam molestiae culpa adipisci voluptate sequi. In
-        voluptates ut dicta quis aliquid tempora eligendi ipsum debitis officiis
-        ratione molestias totam ab consequuntur possimus quibusdam, explicabo,
-        illo laborum sapiente. Rem illo pariatur maxime voluptatum voluptatibus.
-        Fugit eveniet tenetur in obcaecati laudantium odio ipsam adipisci hic
-        sint eligendi pariatur illum veritatis corrupti harum assumenda minus
-        neque, illo omnis voluptatibus laboriosam quos! Omnis eveniet saepe
-        deserunt beatae modi sunt officiis eligendi enim voluptatibus non?
-      </p>
-      {% endblock %}
-
-      <div class="footer">
-        <h3 class="footer__text">Все права защищены &copy;</h3>
-      </div>
-    </div>
-  </body>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Вход</title>
+</head>
+<body>
+    <form action="/login" method="post">
```

Comment:
Чтобы новые функции работали, тут нужно убрать атрибут `action`
```suggestion
    <form method="post">
```